### PR TITLE
Materials UC now shows material limits as well as counts

### PR DESCRIPTION
**Pull request by iainross**
Fixed a couple of incorrect types as well

### CODE REVIEW

File: EDDiscovery/UserControls/UserControlMaterialCommodities.cs [hunk 1]
```diff
@@ -117,7 +117,7 @@ private void Display(MaterialCommoditiesList mcl)
 
                     if (materials)
                     {
-                        rowobj = new[] { m.name, m.shortname, m.category, m.type, m.count.ToString() };
+                        rowobj = new[] { m.name, m.shortname, m.category, m.type, $"{m.count.ToString()}/{MaterialCommodityDB.MaterialLimit(m.type)}" };
```
Comment (robbyxp1) [inline EDDiscovery/UserControls/UserControlMaterialCommodities.cs]: Can't this return null?

**Comment by robbyxp1** [comment]
Can you check my point above and see if i'm right before i pull this in..

**Comment by robbyxp1** [comment]
going to merge anyway

**Comment by iainross** [comment]
Probably can but it won't error just show x/ rather than x/y.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**